enterprise/frontend/src/embedding-sdk/types/theme.ts

```diff
@@ -0,0 +1,10 @@
+import type { MantineThemeOverride } from "@mantine/core";
+
+export type MetabaseTheme = MantineThemeOverride & {
+  other: MetabaseThemeOptions;
+};
+
+/**
+ * Theme options for Metabase components and visualizations.
+ */
+interface MetabaseThemeOptions {}
```

Comment:
This is a stub for now - more custom theme options will be added as they arise, and we should export this type later as well.